src/utils/generateMarkDownTree/generateMarkDownTree.ts

```diff
@@ -54,39 +100,67 @@ export const generateMarkDownTree = (
         // Find the number of '/' chars in the path
         const curDepth = path.match(/\//g)?.length ?? 0;
         const deepestDirName = curDepth
-          ? path.substring(path.lastIndexOf("/") + 1)
+          ? path.substring(path.lastIndexOf('/') + 1)
           : path;
         const longestFileName = getLargestFileNameLengthInLevel(treeCore, 0);
         // Build the current line in the overall output
-        let curLine = "";
-        /* If no '/' chars in path, then this directory is at the root
-    of the project. Don't have to decorate this line of the output with
-    │ or ├── */
-        if (curDepth !== 0) {
-          // For all nested "levels" except for the deepest one, add a │ vertical bar
-          for (let i = 0; i < curDepth - 1; i += 1) {
-            curLine += symbols.vertical;
+        let curLine = '';
+        const spaces = longestFileName - deepestDirName.length;
+        const commentAlignment = comment ? ' '.repeat(spaces) : '';
+        if (isFile === false && core.treeCore.length > 0) {
+          if (
+            curDepth > 0
+            && deepClonedTreeCore
+            && deepClonedTreeCore[0] === core
+          ) {
+            curLine += '</summary><blockquote>';
+          }
+          curLine += `<details><summary>${icon}${hyperLink} ${commentAlignment}${comment}`;
+        } else if (
+          deepClonedTreeCore
+          && core.treeCore.length >= 0
+          && curDepth > 0
```

Comment:
0 represents ROOT?

Reply:
Yup, changed this into const to stay away from magic num